features/0693-credential-representation/README.md

```diff
@@ -0,0 +1,59 @@
+# 0693: Cross-Platform Credential Representation
+
+- Authors: [Horacio Nunez](mailto:horacio.nunez@kiva.org) (Kiva Protocol)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2021-07-06
+- Status Note:
+- Supersedes:
+- Start Date: 2021-02-10
+- Tags: [feature](/tags.md#feature)
+
+## Summary
+Aries Agent developers currently build end user products without a standard method of rendering credentials.
+This RFC proposes how the Aries community can reuse available open technologies to build such a rendering method.
+
+Key results include:
+- Feasibility of cross platform rendering.
+- Enable branding of credentials.
+
+This RFC also enumerate the specific challenges that by using this method could be tackled next.
+
+## Motivation
+The human computer interaction between agents and their users will always gravitate around credentials.
+This interaction is more useful for users when their representation resembles that of their conventional 
+(physical) counterparts.
+
+Achieving effortless semiotic parity with analog credentials doesn't come easy or cheap. 
+In fact, when reviewing new Aries-base projects, is always the case that the rendering of 
+credentials with any form of branding is a demanding portion of the roadmap.
+
+Since the work required here is never declarative the work never stops feeling sysyphean.
+Indeed, the cost of writing code of representing a credential remains constant over time, no
+matter how many times we do it.
+
+Imagine if we achieve declarative while empowering branding.
+
+### Entering SVG
+
+The solution we propose is to adopt SVG as the default format to describe how to represent SSI credentials, and to 
+introduce a convention to ensure that credentials values could be embedded in the final user interface. 
+The following images illustrates how this can work:
+
+![SVG + Credential Values](https://i.imgur.com/3ssaQUB.png "SVG + Credential Values")
+
+### SVG + Credential Values
+
+We propose a notation of the form `{{credential.values.[AttributeName]}}` and `{{credential.names.[AttributeName]}}`.
+This way both values and attributes names can be used in branding activities.
```

Comment:
How would this work with JSON-LD credentials? Does it only support top level values of the `credentialSubject` object (what if it is an array)? Or is this specifically targeted at indy credentials?

Reply:
Yes, we aim to support JSON-LD as well. For non top-level values the path will be is to offer a simple expression language based on F#/Haskell pattern-matching and algebraic types.

Reply:
I was going to ask the same question :-).  Might look at what DIF PE did with JSONPath and then define what that would look like for Indy (which may just work, since it is simple).